### PR TITLE
refactor Promise.race

### PR DESCRIPTION
**Pull request by Jonahss**
@spencerwilson What do you think of this small refactor?

### CODE REVIEW

File: postgres-pubsub.js
Comment (spencerwilson) [inline postgres-pubsub.js:62]: Style nit—although IIRC semicolon style is mixed already. Tangent: Would you be open to adopting Prettier on this project?
```suggestion
    });
```

File: postgres-pubsub.js
Comment (spencerwilson) [inline postgres-pubsub.js]:
```suggestion
    await Promise.race([connectedAndListening, errorThrown]);
```

**Review by spencerwilson — APPROVED** [review]
Better, thanks! 👍 pending adding the missing parens.

**Comment by Jonahss** [comment]
Ah, thanks for those. I'll run tests before merging >.<

Also, sure, we can add a linter.
Did you get a chance to look into CI?

**Comment by spencerwilson** [comment]
> we can add a linter

nit: Prettier isn't a linter but rather a code formatter, JavaScript's version of `go fmt` or `rustfmt` if you know either of those. You're picking up on the fact that Prettier and ESLint do intersect in some ways, it's true; here's a good explanation, if interested: https://prettier.io/docs/en/comparison.html. Prettier came to mind here since it'd ensure semicolon usage (or lack of them) is consistent. Not a huge deal, just a polish thing.

ESLint is also good in that it has rules that could catch bugs.

Haven't looked into CI. I may depending on time.

P.S. If you're considering publishing a release soon, please hold off until we merge a PR to fix #6. I can commit to opening a PR for that soon, and it's pretty valuable to us so would love to get it in the next release if possible.

**Comment by Jonahss** [comment]
Awesome, okay. 
I'll wait for the next PR and then do a release.

Hm yeah, didn't know about Prettier, but sure, I've got no problem with it.

**Comment by spencerwilson** [comment]
Ok, @Jonahss , all my planned work is now up:
- https://github.com/originlabs/graphql-postgres-subscriptions/pull/8
- https://github.com/originlabs/graphql-postgres-subscriptions/pull/9

**Comment by Jonahss** [comment]
Ah, got a CI failure.
Added your CI though :)

File: postgres-pubsub.js
Comment (spencerwilson) [reply postgres-pubsub.js]: @Jonahss Hm, looks like rebasing may have caused this thread to auto-resolve due to referring to a commit that was no longer in the PR branch. My bet is that the CI failure was due to either this bug or is otherwise a preexisting flaky test.

Comment (Jonahss) [reply postgres-pubsub.js]: ah nice. Yeah I rebased. Will fix.

**Comment by Jonahss** [comment]
@spencerwilson merged, and published everything as version 3.0.0 🎊

**Comment by spencerwilson** [comment]
Thanks for the really quick turnaround! My teammates and I appreciate it 😃

**Pull request merged**
